index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="/index.css" rel="stylesheet" />
+
+    <title>Let's Code!!!!!!!</title>
+  </head>
+  <body class=" overflow-hidden " >
+    <div class=" relative  flex p-3 justify-start  bg-img w-full h-screen overflowY-auto " >
+      <header class=" nav fixed bg-blur w-full flex justify-start  items-center -0 absolute-0  " >
+        <div class=" rounded-8  h-full flex items-center justify-start logo  " >
+          <div class=" roundedl-8 logo-bg px-3 flex items-center justify-center  " >
+            <p>Watch.</p>
+          </div>
+          <div class=" flex justify-center items-center px-3 " >
+            <p>#anime</p>
+          </div>
+        </div>
+      </header>
+      <section class=" relative  content px-3 w-full h-screen flex justify-end " >
+        <div class=" fixed absoluteB-0 bg-blur px-3 w-25 rounded-8 h-530 flex flex-col h-screen justify-evenly " >
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer  " >
+            <p>Menu-1</p>
+          </div>
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer " >
+            <p>Menu-2</p>
+          </div>
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer  " >
+            <p>Menu-3</p>
+          </div>
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer " >
+            <p>Menu-4</p>
+          </div>
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer " >
+            <p>Menu-5</p>
+          </div>
+          <div class=" px-3 hover-bg bg-blur rounded-8 w-100 cursor-pointer " >
+            <p>Menu-6</p>
+          </div>
```

Comment:
[Optional] Instead of giving reviews in the comments, you can select specific codes and provide instructive comments. This will help the code assignee understand the specific changes they need to make in particular areas. 😁